deepface/detectors/FastMtcnnWrapper.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+from deepface.detectors import FaceDetector
+
+# Link -> https://github.com/timesler/facenet-pytorch
+# Examples https://www.kaggle.com/timesler/guide-to-mtcnn-in-facenet-pytorch
+
+def build_model():
+    # Optional dependency
+    from facenet_pytorch import MTCNN as fast_mtcnn
```

Comment:
this is optional dependency and one can try to use this detector before installing requirements_additional.txt. would you please add this in a try except block and raise a meaningful message like: you are trying to run an optional detector. you have to install this with that command...

Reply:
Ok, I'll do this rn.
btw, do you prefer we make it non optional instead?

Reply:
please do it optional. deepface is mainly based on tf and including pytorch will make it much bigger.

Reply:
Exactly what I thought.